### PR TITLE
feat: [BREAKING] support mockDingtalkRequest

### PR DESCRIPTION
**Pull request by fengmk2**
only support egg@2 and node-dingtalk@2

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,8 +38,8 @@ exports.dingtalk = {
 };
 
 // {app_root}/app/controller/test.js
-exports.list = function* () {
-  this.body = yield this.app.dingtalk.user.list('1', true);
+exports.list = async (ctx) => {
+  ctx.body = yield ctx.dingtalk.user.list('1', true);
```
Comment (atian25) [inline README.md]: - `async ctx =>`
- `await`

File: config/config.default.js [hunk 2]
```diff
@@ -6,6 +6,7 @@
  * @property {String} corpid - 企业ID
  * @property {String} corpsecret - 企业秘钥
  * @property {String} host - 服务地址
+ * @property {Boolean} enableContextLogger - 是否使用 ctx.logger 记录所有 dingtalk 请求结果
```
Comment (atian25) [inline config/config.default.js:9]: 这个啥场景？

**Review by atian25 — APPROVED** [review]

File: config/config.default.js (hunk 2 above)
Comment (fengmk2) [reply config/config.default.js:9]: https://github.com/ali-sdk/node-dingtalk/pull/28 看这里。需要将钉钉交互的所有日志记录下来，方便排查问题。

**Review by atian25 — COMMENTED** [review]
+1

**Comment by fengmk2** [comment]
2.0.0

**Pull request merged**